checkov/terraform/graph_builder/variable_rendering/renderer.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
Just out of curiosity, what's the purpose of importing `annotations` here? Is it for performance reasons as described in [PEP 563](https://peps.python.org/pep-0563/)?

Reply:
mostly for better and easier type hinting 🙂 you can use built-in types instead of the typing variants, like `list` instead of `List`.